gateway_to_backend/protocol_buffers_files/otap_message.proto

```diff
@@ -18,12 +18,49 @@ enum ScratchpadStatus {
     ERROR = 3;
 }
 
+enum ScratchpadAction {
+    // Will be the default value in case it is later extended
+    UNKNOWN_ACTION = 0;
+    // No propagation, no processing of scratchpad
+    NO_OTAP = 1;
+    // Propagate the target scratchpad but no processing
+    PROPAGATE_ONLY = 2;
+    // Propagate the target scratchpad and process it when available
```

Comment:
when available is a bit confusing. PROCESS immediately or as soon it is received ?